stable/atlantis/templates/statefulset.yaml

```diff
@@ -69,8 +74,8 @@ spec:
           {{- end}}
           args:
             - server
-          {{- if .Values.allowRepoConfig }}
-            - --allow-repo-config
+          {{- if .Values.repoConfig }}
+            - --repo-config=/etc/atlantis/repos.yaml
```

Comment:
Can we set the environment variable `ATLANTIS_REPO_CONFIG` instead? That's how the rest of the args are being passed in.